web-app/django/VIM/settings.py

```diff
@@ -24,6 +24,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = IS_DEVELOPMENT
+# DEBUG = False # uncomment this for testing http error code pages
```

Comment:
Just get rid of this line. If someone wants to set DEBUG to `false`, do it in the environment variables.
```suggestion
```

Reply:
Removed.